lib/gradle-wrapper.ts

```diff
@@ -14,15 +17,19 @@ export function getGradleCommandArgs(targetPath: string): string[] {
   if (targetPath) {
     gradleArgs.push('-p', targetPath);
   }
+  if (initScript) {
+    gradleArgs.push('--init-script', initScript);
+  }
 
   return gradleArgs;
 }
 
 export async function getClassPathFromGradle(
   targetPath: string,
+  initScript: string | null,
```

Comment:
I think it be better to push this argument last, and make all the `initScript` args of type `string?`

Reply:
Done ✅